lib/loader.js

```diff
@@ -37,11 +54,11 @@ function sassLoader(content) {
 
   let resolve = pify(this.resolve);
 
-  // Supported since v4.27.0
-  if (this.getResolve) {
+  // Supported since v4.36.0
+  if (hasGetResolve(self)) {
     resolve = this.getResolve({
-      mainFields: ['sass', 'main'],
-      extensions: ['.scss', '.sass', '.css'],
+      mainFields: ['sass', 'style', '...'],
```

Comment:
may i ask what the `'...'` additions mean (here and the line below)?

also, does this change mean i can use the `"style"` field in `package.json`?